data-prepper-plugins/dynamodb-source/src/test/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/DataFileLoaderTest.java

```diff
@@ -184,23 +169,24 @@ void test_run_loadFile_correctly() throws Exception {
         // Should call s3 getObject
         verify(s3Client).getObject(any(GetObjectRequest.class));
 
-        // Should write to buffer
-        verify(bufferAccumulator, times(total)).add(any(Record.class));
-        verify(bufferAccumulator).flush();
+        verify(exportRecordConverter).writeToBuffer(eq(null), anyList());
 
-        // Should do one last checkpoint when done.
-        verify(coordinator).saveProgressStateForPartition(any(DataFilePartition.class), eq(null));
```

Comment:
The coordinator class is not use in the `DataFileLoader`. This check was here because we weren't mocking the `checkpointer` class. Now we are mocking it so the behavior is tested in the `Checkpointer` unit tests.